src/services/customer.ts

```diff
@@ -56,12 +57,10 @@ export default class CustomerService extends Service {
         return results
     }
 
-    // TODO: Potentially add this at some point
-    // public async listAccessibleCustomers(): Promise<any> {
-    //     const request = new grpc.ListAccessibleCustomersRequest()
-    //     const response = await this.service.listAccessibleCustomers(request)
-    //     console.log(response)
-    // }
+    public async listAccessibleCustomers(): ListAccessibleCustomersResponse {
+        const request = new grpc.ListAccessibleCustomersRequest()
+        return await this.serviceCall('listAccessibleCustomers', request)
```

Comment:
`await` isn't required here when using `return`, so the `async` keyword can also be removed.